app/models/alchemy/image_cropper_settings.rb

```diff
@@ -40,102 +31,51 @@ def [](key)
 
     private
 
+    def ratio
+      return false if fixed_ratio == false
+      return ratio_from_size if fixed_ratio.nil?
+
+      Float(fixed_ratio)
+    end
+
     # Only returns an array of width and height if image is large enough
     # or false to disable min size option of the image cropper
-    def min_size_value(min_size)
-      if image_width >= min_size[:width] && image_height >= min_size[:height]
-        min_size.values
-      else
-        false
-      end
+    def can_be_cropped?
```

Comment:
IMO `large_enough?` is a better name for this, as this does not apply to cropping